src/bundle/Resources/views/content/tabs/versions.html.twig

```diff
@@ -0,0 +1,112 @@
+<div class="ez-versions-list-container">
+    {%  if draftVersions %}
+        <h2 class="ez-list-toolbar-label"><th>{{'locationview.versions.draft.under.edit'|trans({}, 'locationview')}}</th></h2>
+        <table class="ez-table-data">
+            <thead>
+            <tr>
+                <th>{{'locationview.versions.version'|trans({}, 'locationview')}}</th>
+                <th>{{'locationview.versions.modified.language'|trans({}, 'locationview')}}</th>
+                <th>{{'locationview.versions.author'|trans({}, 'locationview')}}</th>
+                <th>{{'locationview.versions.created'|trans({}, 'locationview')}}</th>
+                <th>{{'locationview.versions.last.saved'|trans({}, 'locationview')}}</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for version in draftVersions %}
```

Comment:
I was considering doing this but when we start adding actions to them then they will be different. For example if something is published it has no actions.
If something is Archived it can be deleted etc
Shall I add them to a template now to tidy them up and then we can change at the point when the actions are added if needed?